documentation/processing.tex

```diff
@@ -142,7 +178,7 @@ \subsection{Overview}
 \subsubsection{Statement pasing}
 \label{lab06:proc_stat}
 
-In HLASM language, it is difficult to parse statements into one common structure due to major differences in operand field of different instruction formats. Moreover, when parsing statements, the instruction format can be yet unknown. Therefore, operand fields are stored as strings. This means that during statement passing when instruction format is deduced, the provider has responsibility to produce correct statement format. The following steps are applied in the statement passing (also see \cref{fig06:process_next}):
+In HLASM language, it is difficult to parse statements into one common structure due to its \emph{representational ambiguity}; the major difference between operand fields of different instruction formats. Moreover, when parsing statements, the instruction format can be yet unknown. Therefore, operand fields are stored as strings. This means that during statement passing when instruction format is deduced, the provider has responsibility to produce correct statement format. The following steps are applied in the statement passing (also see \cref{fig06:process_next}):
```

Comment:
representational ambiguity nie je to ked sa rozne veci reprezentuju rovnako?
v zapeti povies ze je to velky rozdiel v operand fileds roznych instrukcii.

passing -> parsing?

Reply:
mirek to navrhol. rozumej, nejednoznačnosť reprezentácie

nie, passing!

Reply:
no ale podla mna nejednoznačnosť reprezentácie nie je rozdiel medzi operand fieldami v roznych instrukciach - vtedy prave tu ambiguitu nemas nie?

Nedava mi zmysel co je to statement passing a ako sa pocas toho moze deducovat instr format. "Pass" poznam len ako predavanie parametrov nejakej funkcii

Reply:
tak vtedy, ked mas rozne formaty instrukcii, mas rozne formaty operandov, teda celkova reprezentacia operandov je nejednoznacna :D asi

Passing som nazval celý process passovania statementu od providera k processorovi. vtedy sa nededucuje instruction format, ale format operandov podla instrukcie, ktora je passnuta processoru

Reply:
no tak snad to pochopia, no.. mne to stale pride akoze si to odporuje

aha tak ked je to je niekde definovane tak ok
inak v nadpise: passing je s dvoma s podla mna